python/tank/authentication/errors.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, url):
         :param str url: Url of the site where login was attempted.
         """
         ShotgunAuthenticationError.__init__(
-            self, "Authentication using username/password is not allowed on the console for an SSO-enabled site."
+            self, "Authentication using username/password is not allowed on the console for %s, an SSO-enabled site." % url
```

Comment:
@manneohrstrom You are much more fluent than me in english, what do you think?

Reply:
@jfboismenu : @mattdaw just read the updated message, and he says its fine.

So I will merge this in feature/sso.